databricks/koalas/frame.py

```diff
@@ -197,6 +197,59 @@ def _reduce_for_stat_function(self, sfun):
         row.name = None
         return row  # Return first row as a Series
 
+    def applymap(self, func):
+        """
+        Apply a function to a Dataframe elementwise.
+
+        This method applies a function that accepts and returns a scalar
+        to every element of a DataFrame.
+
+        .. note:: unlike pandas, it is required for `func` to specify return type hint.
+
+        Parameters
+        ----------
+        func : callable
+            Python function, returns a single value from a single value.
```

Comment:
It is probably worth mentioning that this function must have python 3's style type hints to work, and to refer to the example for what that means (people are not necessarily aware of python 3's typing system).

Reply:
(and that only the return type hint is necessary, not the inputs)

Reply:
There is already a note in the description above, but do we need it here too, or move it to here?
+1 for adding to refer to the example.